### PR TITLE
split lysosomal acid lipase deficiency

### LINKED ISSUE #6379: split term: 'lysosomal acid lipase deficiency'
**Term to split:**
MONDO:0010204 'lysosomal acid lipase deficiency'

**Name for new terms:** Note - when a term should be split, there should be 2+ new terms. Please list the new labels below. 
1. New term (MONDO:0800440) 'lysosomal acid lipase deficiency' (OMIMPS:278000)
2. MONDO:0019149 Cholesteryl ester storage disease (OMIM:278000)

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #6379

### CODE REVIEW

**Comment by nicolevasilevsky** [comment]
@twhetzel 
here is documentation on splitting classes:
https://mondo.readthedocs.io/en/latest/editors-guide/sop_splitting-terms/

Let me know if there is enough detail there to properly review this PR. If not, we should probably update the documentation.

**Comment by nicolevasilevsky** [comment]
related https://github.com/monarch-initiative/mondo/issues/6848

**Review by twhetzel — APPROVED** [review]
A few questions, but approved related to the new term added and obsoleted term.

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -459798,6 +459784,20 @@ is_a: MONDO:0019046 {source="DOID:0060868", source="Orphanet:135"} ! leukodystro
 relationship: has_characteristic MONDO:0021152 ! inherited
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/6216" xsd:anyURI
 
+[Term]
+id: MONDO:0800449
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:459788]: it was originally when I created this issue but then that ID had been issued in the meantime.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -366010,28 +365985,39 @@ xref: SCTID:82500001 {source="MONDO:equivalentTo", source="DOID:14497"}
...
 is_a: MONDO:0003847 {source="MESH:C564736/inferred"} ! hereditary disease
-is_a: MONDO:0010204 {source="DOID:14497", source="Orphanet:75233"} ! lysosomal acid lipase deficiency
-is_a: MONDO:0019245 {source="DOID:14497/inferred", source="MESH:D015223/inferred", source="Orphanet:75233/inferred", source="PMID:21502308"} ! lysosomal lipid storage disorder
+is_a: MONDO:0800449 {source="DOID:14497", source="Orphanet:75233"} ! lysosomal acid lipase deficiency
 relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/mondo/qc-omim-subsumption.sparql
+relationship: excluded_subClassOf MONDO:0002561 {source="DOID:14497/inferred", source="MESH:D015223/inferred", source="MONDO:Redundant", source="NCIT:C61271", source="Orphanet:75233/inferred"} ! lysosomal storage disease
+relationship: excluded_subClassOf MONDO:0019245 {source="DOID:14497/inferred", source="MESH:D015223/inferred", source="Orphanet:75233/inferred", source="PMID:21502308"} ! lysosomal lipid storage disorder
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/5903" xsd:anyURI
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:365993]: I can add that

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -366010,28 +365985,39 @@ xref: SCTID:82500001 {source="MONDO:equivalentTo", source="DOID:14497"}
...
+xref: Orphanet:75234 {source="MONDO:equivalentObsolete"}
 xref: SCTID:57218003 {source="MONDO:equivalentTo", source="DOID:14502"}
 xref: UMLS:C0008384 {source="Orphanet:75234", source="MONDO:equivalentTo", source="DOID:14502"}
-is_a: MONDO:0010204 {source="DOID:14502", source="Orphanet:75234"} ! lysosomal acid lipase deficiency
-property_value: confidence "3.6875" xsd:double
+xref: UMLS:C0043208 {source="OMIM:278000", source="MONDO:ncbi_mim2gene_medline"}
+xref: UMLS:C1416865 {source="OMIM:278000", source="MONDO:notFoundInDiseaseSubset"}
+xref: UMLS:C2936797 {source="Orphanet:275761", source="MONDO:notFoundInDiseaseSubset"}
+is_a: MONDO:0800449 {source="DOID:14502", source="Orphanet:75234"} ! lysosomal acid lipase deficiency
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/6617 {source="OMIM:278000"} ! LIPA
```
[the remark this answers is not in the document]
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:366020]: I can add that

**Pull request merged**